src/app/Http/Controllers/SupportPageController.php

```diff
@@ -39,15 +39,20 @@ public function show(): View
             );
         }
 
+        $build = base_path();
+        $stringPosition = strrpos($build, '\\');
```

Comment:
Use `DIRECTORY_SEPARATOR` constant; this will fail on Linux paths as they use `/`.

Reply:
`$index` is a better variable name.